source/sdk/math.ooc

```diff
@@ -74,22 +70,8 @@ extend Int {
 		result % divisor
 	}
 	clamp: func (floor: This, ceiling: This) -> This {
-		if (this > ceiling)
-			ceiling
-		else if (this < floor)
-			floor
-		else
-			this
+		this > ceiling ? ceiling : (this < floor ? floor : this)
```

Comment:
I don't think it's too bad, and having two `?:`-operators is nothing new. Do a regex search for `.*\?.*: \(.*\?.*:.*\)`.